__tests__/app.test.js

```diff
@@ -139,18 +177,63 @@ describe("GET /api/reviews/:review_id", () => {
   })
   test("responds with 404 upon being given a valid but non-existent id", () => {
     return request(app)
-    .get('/api/reviews/26')
-    .expect(404)
-    .then(({ body }) => {
-      expect(body.msg).toBe("error: review id not found");
-    })
-  })
+      .get("/api/reviews/26")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("error: review id not found");
+      });
+  });
   test("responds with 400 when given an invalid id", () => {
     return request(app)
-    .get('/api/reviews/a')
-    .expect(400)
-    .then(({ body }) => {
-      expect(body.msg).toBe("error: invalid input");
-    })
-  })
-})
\ No newline at end of file
+      .get("/api/reviews/a")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("error: invalid input");
+      });
+  });
+});
+
+describe.only("GET /api/reviews/:review_id/comments", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/reviews/2/comments").expect(200);
+  });
+  test("responds with an object with a key of comments containing an array", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+      });
+  });
+  test("array from response contains comments with appropriate properties", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            review_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("if given a valid review id that doesn't have any comments, respond with 404 and an appropriate message", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(404)
```

Comment:
This should be a 200. Nothing has gone wrong, it should return an empty array showing the user that there is no comments for that valid review 👍